manifests/params.pp

```diff
@@ -22,13 +22,19 @@
 #   The oauth key for Candlepin
 # @param candlepin_oauth_secret
 #   The oauth secret for Candlepin
+# @param scl_postgresql
+#   The scl name for PostgreSQL
+# @param postgresql_evr_package
+#   The contextual package name for the PostgreSQL EVR extension
 class katello::params (
   Stdlib::HTTPSUrl $pulp_url = "https://${facts['fqdn']}/pulp/api/v2/",
   Stdlib::HTTPSUrl $crane_url = "https://${facts['fqdn']}:5000",
   Stdlib::Host $qpid_hostname = 'localhost',
   String[1] $candlepin_oauth_key = $katello::globals::candlepin_oauth_key,
   String[1] $candlepin_oauth_secret = $katello::globals::candlepin_oauth_secret,
   Stdlib::Host $candlepin_host = 'localhost',
-  Stdlib::HTTPSUrl $candlepin_url = "https://${candlepin_host}:8443/candlepin"
+  Stdlib::HTTPSUrl $candlepin_url = "https://${candlepin_host}:8443/candlepin",
+  String[1] $scl_postgresql = 'rh-postgresql12',
+  String[1] $postgresql_evr_package = "postgresql-evr-0.0.2-1.el${$::os['release']['major']}.x86_64"
```

Comment:
i think we'd want just the package name here, and then use  :ensure => latest  in the package delcaration above (unless i'm completely off base here)

Reply:
Can I have the shortened package name here and still pick the right release (el7 vs el8)?

Reply:
yeah, its basically doing a 'yum install $PKG_NAME'  so as long as its doing   'rh-postgresql12-postgresql-evr' on el7 and 'postgresql-evr' on el8, i think we're good

Reply:
makes sense, I'll update